rasa_utils/bot_server_channel.py

```diff
@@ -98,95 +98,102 @@ def __init__(
         self.cors_origins = [u'*']
         self.agent = agent
         self.port = port
+        self.preprocessor = preprocessor
 
-    @app.route("/conversations/<cid>/log", methods=["GET"])
-    @check_cors
-    def show_log(self, request, cid):
-        request.setHeader("Content-Type", "application/json")
-        return json.dumps(self.message_store[cid])
-
-    @app.route("/conversations/<cid>/tracker", methods=["GET"])
-    @check_cors
-    def tracker(self, request, cid):
-        tracker = self.agent.tracker_store.get_or_create_tracker(cid)
-        tracker_state = tracker.current_state(
-            should_include_events=True,
-            only_events_after_latest_restart=True
-        )
+    @classmethod
+    def name(cls):
+        return "chatroom"
+
+    def blueprint(self, on_new_message):
+        bot_server_webhook = Blueprint('bot_server_webhook', __name__)
+        CORS(bot_server_webhook)
 
-        request.setHeader("Content-Type", "application/json")
-        return json.dumps(tracker_state)
+        @bot_server_webhook.route("/", methods=['GET'])
+        def static():
+            if self.static_files is None:
+                return NoResource()
```

Comment:
I just noticed one more thing during testing. `NoResource` will crash the server. That's a `Klein` method and is incompatible with Flask. Either remove the `static` method completely or replace `NoResource` and `File` with Flask compatible versions.

Please adjust the import accordingly.